docs/kqueen.cli_api_examples.rst

```diff
@@ -1,15 +1,18 @@
-Kqueen CLI API Examples
+KQueen CLI API examples
 -----------------------
 
-Obtain bearer token and authenticate to kqueen:
+This section lists the available KQueen API operations that you can perform
+through the command-line interface and provides examples.
+
+**To obtain a bearer token and authenticate to KQueen:**
 
 .. code-block:: bash
 
    $ TOKEN=$(curl -s -H "Content-Type: application/json" --data '{ "username": "admin", "password": "default" }' -X POST 127.0.0.1:5000/api/v1/auth | jq -r '.access_token')
```

Comment:
Maybe it makes sense to convert long bash commands into multiline ones (maybe in a separate PR). Because it's not comfortable to scroll the code horizontally